documentation/staging/content/samples/simple/domains/domain-home-in-image/_index.md

```diff
@@ -60,9 +61,21 @@ The script will perform the following steps:
 
 * Create a directory for the generated properties and Kubernetes YAML files for this domain if it does not already exist.  The pathname is `/<path to output-directory>/weblogic-domains/<domainUID>`. If the directory already exists, its contents will be removed.
 * Create a properties file, `domain.properties`, in the directory that is created above. This properties file will be used to create a sample WebLogic Server domain.
-* Clone the WebLogic docker-images project into the directory that is derived from the `domainHomeImageBuildPath` property using `git clone https://github.com/oracle/docker-images.git`. By default, the script always cleans up the directory and clones it again on every run. You need to specify the `-k` option if you want to use a previously cloned project. Note that if the specified `domainHomeImageBuildPath` is empty, the script will still clone the project even if the `-k` option is specified.
-* Replace the built-in user name and password in the `properties/docker-build/domain_security.properties` file with the `username` and `password` that are supplied on the command line using the `-u` and `-p` options. These credentials need to match the WebLogic domain administrator credentials in the secret that is specified using the `weblogicCredentialsSecretName` property in the `create-domain-inputs.yaml` file.
-* Build a container image based on the Docker sample, [Example Image with a WebLogic Server Domain using the Oracle WebLogic Scripting Tooling (WLST)](https://github.com/oracle/docker-images/tree/master/OracleWebLogic/samples/12213-domain-home-in-image) or [Example Image with a WebLogic Server Domain using the Oracle WebLogic Deploy Tooling (WDT)](https://github.com/oracle/docker-images/tree/master/OracleWebLogic/samples/12213-domain-home-in-image-wdt). It will create a sample WebLogic Server domain in a container image.
+* Download the latest [WebLogic Deploy Tooling](https://github.com/oracle/weblogic-deploy-tooling) (WDT) and [WebLogic Image Tool](https://github.com/oracle/weblogic-image-tool) (WIT) installer ZIP files to your `/tmp/dhii-sample/tools` directory. Both WDT and WIT are required to create your Model in Image container images. 
+  Visit the GitHub [WebLogic Deploy Tooling Releases](https://github.com/oracle/weblogic-deploy-tooling/releases) and [WebLogic Image Tool Releases](https://github.com/oracle/weblogic-image-tool/releases) web pages to determine the latest release version for each.
+
+* Set up the WebLogic Image Tool to the `/tmp/dhii-sample/tools/imagetool` directory. Set the
+  WIT cache store location to the `/tmp/dhii-sample/tools/imagetool-cache` directory and
+  put a `wdt_<WDT_VERSION>` entry in the tool's cache which points to the path of the WDT ZIP file installer.
```

Comment:
Should we use /tmp as the working directory?  Some system will wipe clean the /tmp if it is restarted.  In that case, user will lose everything.

Reply:
we choose a tmp directory so that it will likely to get cleaned up after running the samples. It is configurable. Should we modify the doc here to emphasize that it is configurable, such as:
Set up the WebLogic Image Tool to the `<tools>/imagetool` directory, where `<tools>` is the directory specified in the `toolsDir` parameter in the inputs YAML file.

Reply:
As long as it is configurable, that's fine, document where to change is good.    Also, do we check or document space needed for the sample?

Reply:
I don't think it is currently documented

Reply:
added comma between cache and which